contracts/token/Minter.sol

```diff
@@ -76,12 +104,33 @@ contract Minter is Manager, IMinter {
         ParameterUpdate("inflationChange");
     }
 
-    function transferTokenOwnership(address _newOwner) external onlyControllerOwner {
-        livepeerToken().transferOwnership(_newOwner);
-    }
-
-    /*
-     * @dev Create reward based on a fractional portion of the mintable tokens and redistributable funds for a round
+    /**
+     * @dev Migrate to a new Minter by transferring ownership of the token as well
+     * as the current Minter's token balance to the new Minter
+     * @param _newMinter Address of new Minter
+     */
+    function migrateToNewMinter(IMinter _newMinter) external onlyControllerOwner {
+        // New Minter cannot be the current Minter
+        require(_newMinter != this);
+        // Check for null address
+        require(address(_newMinter) != address(0));
+
+        IController newMinterController = _newMinter.getController();
+        // New Minter must have same Controller as current Minter
+        require(newMinterController == controller);
```

Comment:
These next two checks don't actually guarantee that the contract address being passed into `migrateToNewMinter` is a valid Minter contract, but more so just serve as sanity checks to make sure that the provided address at least implements the `getController` function of the Minter interface, the Controller returned matches the Controller of the old Minter and that the Controller still has the old Minter registered (might be debatable how meaningful this last check really is...)